.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Delete this file. Maybe we should add `.gitginore`.

Reply:
the `.vscode` is in `.gitignore`... 
anyway i remoted the vscode launch